lib/utils/razorpay-utils.d.ts

```diff
@@ -0,0 +1,44 @@
+export interface IRazorpayWebhook {
+    order_id?: string;
+    payment_id?: string;
+    subscription_id?: string;
+    payment_link_id?: string;
+    payment_link_reference_id?: string;
+    payment_link_status?: string;
+}
+
+/**
+* Verify webhook verification
+*
+* @param {string} body 
+* raw webhook request body
+* @param {string} signature  
+* The hash signature is calculated using HMAC with SHA256 algorithm; with your webhook 
+* secret set as the key and the webhook request body as the message.
+* @param {string} secret
+* your webhook secret
+*
+*/
+export function validateWebhookSignature(body: string, signature: string, secret: string): boolean
+
+/**
+*  Payment verfication
+*
+* @param {object} payload
+* Check [doc](https://github.com/razorpay/razorpay-node/blob/master/documents/paymentVerfication.md) for required params
+* @param {string} signature
+* The hash signature is calculated using HMAC with SHA256 algorithm; with your webhook 
+* secret set as the key and the webhook request body as the message.
+* @param {string} secret
+* your webhook secret
+*
+*/
+export function validatePaymentVerification(payload: IRazorpayWebhook, signature: string, secret: string): boolean
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can we use union types here  for the 3 different cases

Reply:
Used overloading here